web/src/components/TaskProgressCheckbox.test.tsx

```diff
@@ -234,6 +234,48 @@ describe("<TaskProgressCheckbox />", () => {
       });
     });
 
+    it("redirects with fromTaxRegistration as false when checking TaskProgressCheckbox as Foreign Remote Seller", async () => {
+      const foreignRemoteSeller = generateProfileData({
+        businessPersona: "FOREIGN",
+        foreignBusinessTypeIds: ["revenueInNJ", "transactionsInNJ"],
+      });
+      renderTaskCheckbox(taxTaskId, generateBusiness({ profileData: foreignRemoteSeller }));
+      await selectCompleted();
+      expect(currentBusiness().taskProgress[taxTaskId]).toEqual("COMPLETED");
+      expect(mockPush).toHaveBeenCalledWith({
+        pathname: ROUTES.dashboard,
+        query: { fromFormBusinessEntity: "false", fromTaxRegistration: "false" },
+      });
+    });
+
+    it("redirects with fromTaxRegistration as false when checking TaskProgressCheckbox as Foreign Remote Worker", async () => {
+      const foreignRemoteWorker = generateProfileData({
+        businessPersona: "FOREIGN",
+        foreignBusinessTypeIds: ["employeesInNJ"],
+      });
+      renderTaskCheckbox(taxTaskId, generateBusiness({ profileData: foreignRemoteWorker }));
+      await selectCompleted();
+      expect(currentBusiness().taskProgress[taxTaskId]).toEqual("COMPLETED");
+      expect(mockPush).toHaveBeenCalledWith({
+        pathname: ROUTES.dashboard,
+        query: { fromFormBusinessEntity: "false", fromTaxRegistration: "false" },
+      });
+    });
+
+    it("redirects with fromTaxRegistration as true when checking TaskProgressCheckbox as dakota nexus", async () => {
```

Comment:
[pebble] This test doesn't cover the logic we implemented with the new feature. We can delete this.

Reply:
I think it would be good to have anyways. 

Before this we didn't have any test covering the "default case" of showing the certification snackbar from Tax Redirect. 

It's also great to have a positive case here to make sure its possible (if the alert was for some reason always disabled in our code we would want our tests to cover that)

Reply:
We could try and make this more generic? Perhaps as the snackbar is suppose to show for all other cases coming from the tax registration page. Or more robust by doing all the various persona but I didn't think it was super necessary here.